### PR TITLE
Add admin button to go to members' list in manager settings (Fixes #6740)

### LINKED ISSUE #6740: UserProfileDialog: Admin view Profile option
Would it be a good idea to add a button to allow an admin to view the user full profile in the manager settings?

### PR DESCRIPTION
**Pull request by Mutugiii**
Fixes #6740 

Add a button to allow admins to go to the members' list in manager settings.

### CODE REVIEW

File: src/app/users/users-profile/users-profile-dialog.component.ts [hunk 1]
```diff
@@ -18,10 +20,15 @@ export class UserProfileDialogComponent {
...
+  constructor(
+  @Inject(MAT_DIALOG_DATA) public data: any,
+  private userService: UserService
+  ) {
     this.name = data.member.name;
     this.planetCode = data.member.userPlanetCode;
+    const currentUser = this.userService.get();
+    this.isHidden = !currentUser.isUserAdmin;
   }
-
```
Comment (paulbert) [inline src/app/users/users-profile/users-profile-dialog.component.ts:26]: Please leave this space in.

File: src/app/users/users-profile/users-profile-dialog.component.ts [hunk 2]
```diff
@@ -18,10 +20,15 @@ export class UserProfileDialogComponent {
 
   name: string;
   planetCode: string;
+  isHidden: boolean;
 
-  constructor(@Inject(MAT_DIALOG_DATA) public data: any) {
+  constructor(
+  @Inject(MAT_DIALOG_DATA) public data: any,
```
Comment (paulbert) [inline src/app/users/users-profile/users-profile-dialog.component.ts:26]: These two lines should be indented two more spaces.

File: src/app/users/users-profile/users-profile-dialog.component.ts [hunk 3]
```diff
@@ -10,6 +11,7 @@ import { MAT_DIALOG_DATA } from '@angular/material';
       <planet-users-profile [isDialog]="true" [userName]="name" [planetCode]="planetCode"></planet-users-profile>
     </mat-dialog-content>
     <mat-dialog-actions>
+      <button color="primary" *ngIf="!isHidden" routerLink="/manager/users" mat-raised-button mat-dialog-close i18n>Manager Settings</button>
```
Comment (paulbert) [inline src/app/users/users-profile/users-profile-dialog.component.ts:14]: This link should go to the specific member profile, not the list of all members.

File: src/app/users/users-profile/users-profile-dialog.component.ts [hunk 4]
```diff
@@ -18,10 +20,15 @@ export class UserProfileDialogComponent {
...
 
-  constructor(@Inject(MAT_DIALOG_DATA) public data: any) {
+  constructor(
+  @Inject(MAT_DIALOG_DATA) public data: any,
+  private userService: UserService
+  ) {
     this.name = data.member.name;
     this.planetCode = data.member.userPlanetCode;
+    const currentUser = this.userService.get();
+    this.isHidden = !currentUser.isUserAdmin;
```
Comment (paulbert) [inline src/app/users/users-profile/users-profile-dialog.component.ts:32]: Please use [ViewChild](https://angular.io/api/core/ViewChild) to get the `editable` property from `UsersProfileComponent`.  That is based on some more conditions in addition to `isUserAdmin` which should also be used to show/hide the link to the user profile page.

Comment (Mutugiii) [reply src/app/users/users-profile/users-profile-dialog.component.ts]: I cannot seem to get it working using editable property working, the state of editable seems to be always `false` even in the profileView function in the users-profile.component.ts.

**Pull request merged**